static/main.js

```diff
@@ -30,6 +30,30 @@ function init() {
         macroString += "&" + match[0].substr(1);
     }
 
+    // The `before` search parameter puts normal text before the math.
+    // The `after` search parameter puts normal text after the math.
+    // Example use: testing baseline alignment.
+    if (/(?:^\?|&)(?:before|after)=/.test(window.location.search)) {
+        var mathContainer = math;
+        mathContainer.id = "math-container";
+
+        if ((match = /(?:^\?|&)before=([^&]*)/.exec(window.location.search))) {
+            var child = document.createTextNode(decodeURIComponent(match[1]));
+            mathContainer.appendChild(child);
+            macroString += "&" + match[0].substr(1);
+        }
+
+        math = document.createElement("SPAN");
```

Comment:
As much as I like capital tags personally, I think `"span"` would be more consistent with other KaTeX code (e.g. `src/domTree.js`).